### PR TITLE
Expose app balance record fields in v2 account model

### PR DESCRIPTION
**Pull request by pzbitskiy**

### CODE REVIEW

**Comment by pzbitskiy** [comment]
@winder

File: daemon/algod/api/algod.oas2.json [hunk 1]
```diff
@@ -815,6 +840,163 @@
...
+      "properties": {
+        "key": {
+          "type": "string"
+        },
+        "value": {
+          "$ref": "#/definitions/ApplicationValue"
+        }
+      }
+    },
+    "ApplicationValue": {
```
Comment (winder) [inline daemon/algod/api/algod.oas2.json:924]: I'm not following what this is, exactly. There's an enum format that can be applied to strings if thats what you're trying to get.

File: libgoal/libgoal.go [hunk 2]
```diff
@@ -649,6 +650,160 @@ func (c *Client) AccountInformation(account string) (resp v1.Account, err error)
...
+func (c *Client) AccountInformationV2(account string) (resp generatedV2.Account, err error) {
+	algod, err := c.ensureAlgodClient()
+	if err == nil {
+		resp, err = algod.AccountInformationV2(account)
+	}
+	return
+}
+
+// ConvertAccountInformation converts generatedV2.Account to basics.AccountData
+func ConvertAccountInformation(a *generatedV2.Account) (basics.AccountData, error) {
```
[the remark this answers is not in the document]
Comment (pzbitskiy) [reply libgoal/libgoal.go:663]: this is a really good idea, thank you

File: daemon/algod/api/algod.oas2.json (hunk 1 above)
Comment (pzbitskiy) [reply daemon/algod/api/algod.oas2.json:924]: this is one-to-one mapping for basics.TealValue

File: libgoal/libgoal.go [hunk 3]
```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/algorand/go-algorand/config"
 	"github.com/algorand/go-algorand/crypto"
 	"github.com/algorand/go-algorand/daemon/algod/api/spec/common"
-	"github.com/algorand/go-algorand/daemon/algod/api/spec/v1"
+	v1 "github.com/algorand/go-algorand/daemon/algod/api/spec/v1"
```
Comment (justicz) [inline libgoal/libgoal.go:31]: is this necessary?

Comment (pzbitskiy) [reply libgoal/libgoal.go:31]: this is done by vscode's go linter on save, and I did not find a way to disable this check

**Pull request merged**